<launch>
	<node pkg="manyears_ros" type="manyears_ros" name="many_ears" output="screen"> <!--launch-prefix="xterm -e gdb -\-args" -->
		<param name="use_audio_stream" value="true"/>
		<param name="iterative_path" value="$(find manyears_ros)/data/part/"/>
		<param name="iterative_delay" value="1"/>
		<param name="iterative_enable" value="false"/>
		<param name="raw_file" value="$(find manyears_ros)/data/debord.raw" />
		<param name="config_file" value="$(find manyears_ros)/data/8sounds_cube.mes" />
		<param name="enable_separation" value="true" />
		<param name="save_audio_file" value="$(find manyears_ros)/data/recorded.raw" />
		<remap from="stream"  to="/audio_stream"/>
	</node>
</launch>
